client/src/pages/EditProfile/EditProfileForm/DateSelector.tsx

```diff
@@ -0,0 +1,140 @@
+import { FormControl, Grid, MenuItem, InputLabel, Typography, Select } from '@mui/material';
+import { useEffect, useState } from 'react';
+import useInputState from './useInputState';
+const DateSelector = (): JSX.Element => {
```

Comment:
👍🏽 

Note: Maybe include a line space here to separate the imports and the component